benchmarks/sum/lua.lua

```diff
@@ -0,0 +1,11 @@
+local _M = {}
+function _M.sum(arr, n)
+    local s = 0.0
+    for i = 1, n do
```

Comment:
Shadowing the "i" is a bit confusing. What about `for rep = 1, nrep`?

Reply:
Done